packages/utils/src/lib/reports/utils.ts

```diff
@@ -71,7 +71,7 @@ export function countCategoryAudits(
   // Create lookup object for groups within each plugin
   const groupLookup = plugins.reduce<Record<string, Record<string, Group>>>(
     (lookup, plugin) => {
-      if (plugin.groups.length === 0) {
+      if (plugin.groups == null || plugin.groups.length === 0) {
```

Comment:
Not saying this is better, but it is shorter. Feel free to ignore :grin: 

```suggestion
      if (!plugin.groups?.length) {
```

Reply:
I like it the original way more as it explicitly states there are two valid cases in which it should end early.